draft-ietf-ppm-dap.md

```diff
@@ -2856,8 +2875,8 @@ corresponding media types types:
 - AggregationJobContinueReq {{aggregation-leader-continuation}}: "application/dap-aggregation-job-continue-req"
 - AggregateShareReq {{collect-flow}}: "application/dap-aggregate-share-req"
 - AggregateShare {{collect-flow}}: "application/dap-aggregate-share"
-- CollectionReq {{collect-flow}}: "application/dap-collect-req"
-- Collection {{collect-flow}}: "application/dap-collection"
+- CollectionJobReq {{collect-flow}}: "application/dap-collection-job-req"
+- CollectionJobResp {{collect-flow}}: "application/dap-collection-job-resp"
```

Comment:
Likewise here: my preference would be to not change the media type, even if we change the name of the message (which in itself is not a wire breaking change).

Reply:
Ignore this comment (see above)